src/Files.App/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -67,42 +65,27 @@ public FolderLayoutModes LayoutMode
             set
             {
                 if (SetProperty(ref LayoutPreference.LayoutMode, value, nameof(LayoutMode)))
-                {
                     LayoutPreferencesUpdateRequired?.Invoke(this, new LayoutPreferenceEventArgs(LayoutPreference));
-                }
             }
         }
 
         public uint GetIconSize()
         {
             if (LayoutMode == FolderLayoutModes.DetailsView)
-            {
                 return Constants.Browser.DetailsLayoutBrowser.DetailsViewSize; // ListView thumbnail
-            }
+
             if (LayoutMode == FolderLayoutModes.ColumnView)
```

Comment:
Would a switch case work here?

Reply:
We can't since not all ifs use `==`.

Reply:
I mean, I'm not sure a 3-cases switch would lead to performance improvements

Reply:
It helps with code readability but no need to worry if it doesn't work here

Reply:
It would look something like this:
![1](https://user-images.githubusercontent.com/102259289/193684885-3aea3bc6-6635-4144-b067-e4b9068fd9b2.png)

Reply:
I think that's cleaner but let's keep as is for now.